pytorch_toolkit/face_recognition/train.py

```diff
@@ -90,7 +95,16 @@ def train(args):
 
     model = models_backbones[args.model](embedding_size=args.embed_size,
                                          num_classes=dataset.get_num_classes(), feature=False)
+
+    set_dropout_fn = model.set_dropout_ratio
```

Comment:
It is needed for training compressed models without dropout

Reply:
I mean that why dont use 
`model.set_dropout_ratio(0.1)`
insdead of 
```
set_dropout_fn = model.set_dropout_ratio
...
set_dropout_fn(0.1)
```

Reply:
Because if we compress model, we wrap it with compession_algo
` model = compression_algo.model (106 line)` 
and should call model.model.set_dropout_fn(0.1). But if we train simple model, we should call model.set_dropout_fn(0.1).
This hint is used only to write universal code for several conditions.

Reply:
Hm, ok.